server/src/controllers/job_lead/getAllJobLeads.js

```diff
@@ -1,14 +1,138 @@
 const logger = require("pino")();
+const { Op, literal, Sequelize } = require("sequelize");
 const JobLead = require("../../models/job_lead.model");
 
 const getAllJobLeadsRequestHandler = async (req, res) => {
   try {
-    const job_leads = await JobLead.findAll();
+    const page = req?.query?.page ? parseInt(req.query.page, 10) : 0;
```

Comment:
I would recommend changing this to make the first page "page 1" rather than "page 0"

Reply:
Hey Daniel, by default MUI datagrids index at page 0. so if i set the page to be 1 then the data grid will force onto the second page.

Reply:
Ok then yeah its fine to leave it as it is